client/core/bookie.go

```diff
@@ -34,7 +34,7 @@ type BookFeed struct {
 // feed when it's no longer being used.
 func NewBookFeed(close func(feed *BookFeed)) *BookFeed {
 	return &BookFeed{
-		C:     make(chan *BookUpdate, 1),
+		C:     make(chan *BookUpdate, 256),
```

Comment:
Did you experience a deadlock on a send or is this a performance consideration?

Reply:
Whoops. Never submitted my self-review.